js/damage.js

```diff
@@ -211,7 +211,7 @@ function getDamageResult(attacker, defender, move, field) {
 
 	description.HPEVs = defender.HPEVs + " HP";
 
-	if (move.name === "Seismic Toss" || move.name === "Night Shade") {
+	if (["Seismic Toss","Night Shade"].indexOf(move.name) !== -1) {
```

Comment:
Does `Array#indexOf` exist on older browsers? You may want to polyfill if you haven't.

Reply:
According to Mozilla Developer Network, this method is natively supported since IE9. Does anyone use an older browser that IE9, out of curiosity?

Reply:
Usually, the problem isn't IE9, it's Safari 5.

Also, why not support IE8? Polyfilling isn't a big deal; client does it.